crates/store/re_log_encoding/src/encoder.rs

```diff
@@ -120,15 +179,20 @@ impl<W: std::io::Write> Encoder<W> {
         }
     }
 
+    // NOTE: This cannot be done in a `Drop` implementation because of `Self::into_inner` which
+    // does a partial move.
+    #[inline]
     pub fn finish(&mut self) -> Result<(), EncodeError> {
         MessageHeader::EndOfStream.encode(&mut self.write)?;
         Ok(())
```

Comment:
We could warn in Drop if this hasn't been called, at least in debug builds

Reply:
Not sure what you mean? We cannot implement `Drop` for `Encoder`?